content/en/blog/2024/collecting-file-based-java-logs-with-opentelemetry.md

```diff
@@ -0,0 +1,234 @@
+Sending Java logs with OpenTelemetry typically uses the OTLP protocol. However, some scenarios require logs to be output to files or stdout due to organizational or reliability needs.
```

Comment:
- file should be called `index.md` in the folder
- add metadata similar to here https://github.com/open-telemetry/opentelemetry.io/blame/12fb8568070bd0b01282c6a738f6dde84aef1b77/content/en/blog/2024/spring-starter-stable/index.md#L1-L12
  - issue: https://github.com/open-telemetry/opentelemetry-specification/pull/4183
  - sig: Java, Specification
  - date: leave empty utlil we get a date